### ISSUE TITLE
We should run the whole game at 30fps

### ISSUE THREAD

**Issue by ldunnplaymint** [playmint/ds#730]
It seems we are not enforcing any framerate to any device. The game doesn't need a high fps to play so we should just enforce 30fps everywhere.

**Comment by chrisfarms** [comment]
The situation is more complicated than simply capping framerate on the Unity side for WebGL than for a "normal" build.

The _browser_ must decide the framerate itself for the best performance. Browsers automatically decide the rate of requestAnimationFrame (based on things like screen, gpu, cpu capability) and then have a mechanism for dropping and merging frames. Capping frame rate on the unity side sidesteps this mechanism and causes freezes, lags and generally poor performance while it attempts to use timers to sync things up, which will rarely sync up to browser's framerate and act funny.

The problem as I understand it at the moment, is that sometimes the browser's decided framerate is unsuitable (like a retina screen on a lower power macbook air wanting to drive at 120fps) .... it might be that in this case, the poor sideeffects of the out of sync frames are perferable to the high cpu of attempting 120fps ... but we wouldn't want that for all devices 

Basically we should understand the problem more and see if we can work out why the browser thinks it can handle 120fps!
